packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -322,36 +324,53 @@ export function makeRumPublicApi(
          *
          * @param name name of the custom vital
          * @param options.context custom context attached to the vital
-         * @param options.startTime epoch timestamp of the start of the custom vital (if not set, will use current time)
+         * @param options.details  Details of the vital.
          */
         ;(rumPublicApi as any).startDurationVital = monitor(
-          (name: string, options?: { context?: object; startTime?: number }) => {
-            strategy.startDurationVital({
+          (name: string, options?: { context?: object; details?: string }) => {
+            addTelemetryUsage({ feature: 'start-duration-vital' })
+            return strategy.startDurationVital({
               name: sanitize(name)!,
-              startClocks: options?.startTime ? timeStampToClocks(options.startTime as TimeStamp) : clocksNow(),
               context: sanitize(options?.context) as Context,
+              details: sanitize(options?.details) as string | undefined,
             })
-            addTelemetryUsage({ feature: 'start-duration-vital' })
           }
         )
 
         /**
-         * Stop a custom duration vital
+         * Add a custom duration vital
          * stored in @vital.custom.<name>
          *
          * @param name name of the custom vital
+         * @param options.startTime epoch timestamp of the start of the custom vital
+         * @param options.duration duration of the custom vital
          * @param options.context custom context attached to the vital
-         * @param options.stopTime epoch timestamp of the stop of the custom vital (if not set, will use current time)
+         * @param options.details  Details of the vital.
          */
-        ;(rumPublicApi as any).stopDurationVital = monitor(
-          (name: string, options?: { context?: object; stopTime?: number }) => {
-            strategy.stopDurationVital({
+        ;(rumPublicApi as any).addDurationVital = monitor(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** To avoid confusion `startDurationVital`, ‏we could add a warning if the no `options` is passed. We could say `You might want to us the startVitalDuration API ..`. I'm saying this because it happens to me :) I used addDurationVital instead of startDurationVital and it silently failed.